Cargo.toml

```
@@ -5,6 +5,7 @@ members = [
     "boa_wasm",
     "boa_tester",
     "boa_unicode",
+    "boa_examples",
```

```suggestion
    "examples",
```
I think this line needs to have the name of the directory at the top of the repository. The other option is changing the name of the directory from `examples` to `boa_examples`.